helpers/browser/driver.js

```diff
@@ -110,12 +116,17 @@ class ChromeProtocol {
         if (err) {
           return reject(result);
         }
-
         resolve(result);
       });
     });
   }
 
+  pendingCommandsComplete() {
```

Comment:
Yup. [It's](https://github.com/cyrus-and/chrome-remote-interface#event-ready) a convenience API that cri provides.

> Emitted every time that there are no more pending commands waiting for a response from Chrome. Note that the interaction with Chrome is asynchronous so the only way to serialize a sequence of commands is to use the callback provided by the chrome.send method. This event acts as a barrier and it is useful to avoid the callback hell in certain simple situations.